config/default/manager_default_images.yaml

```diff
@@ -17,3 +17,6 @@ spec:
           value: quay.io/podified-antelope-centos9/openstack-memcached:current-podified
         - name: INFRA_REDIS_IMAGE_URL_DEFAULT
           value: registry.redhat.io/rhel9/redis-6:latest
+        # TODO create its own container image, instead of using neutron one
+        - name: INFRA_DNSMASQ_IMAGE_URL_DEFAULT
+          value: quay.io/tripleozedcentos9/openstack-neutron-server:current-tripleo
```

Comment:
Should we use `quay.io/podified-antelope-centos9/openstack-neutron-server:current-podified` since is it available?

Reply:
I guess if we're just going to change this to a dedicated `dnsmasq` in the near future, this might be okay for now.  I just know they're trying to get rid of the `tripleozedcentos9` images.

Reply:
yes, just pushed a fix for that